md2conf/converter.py

```diff
@@ -270,9 +265,10 @@ def _transform_link(self, anchor: ET._Element) -> None:
         # convert the relative URL to absolute URL based on the base path value, then look up
         # the absolute path in the page metadata dictionary to discover the relative path
         # within Confluence that should be used
-        absolute_path = os.path.abspath(os.path.join(self.base_path, relative_url.path))
-        if not absolute_path.startswith(self.base_path):
-            msg = f"relative URL points to outside base path: {url}"
+        absolute_path = (self.base_path / relative_url.path).absolute()
+        # TODO: Looks bit suspicious, should look in path.resolve for better checking
```

Comment:
This is only meant as a rather simple simple check to ensure we don't export paths that don't make sense in Confluence, e.g. `../../image.png` would be illegal in Confluence because there is no parent folder.

Reply:
I see...  so fixed this too